src/Radarr.Api.V3/Collections/CollectionController.cs

```diff
@@ -145,6 +145,11 @@ private IEnumerable<CollectionResource> MapToResource(List<MovieCollection> coll
                     var movieResource = movie.ToResource();
                     movieResource.Folder = _fileNameBuilder.GetMovieFolder(new Movie { MovieMetadata = movie }, namingConfig);
 
+                    if (_movieService.FindByTmdbId(movie.TmdbId) == null)
```

Comment:
This is going to result in thousands of db calls for a collection endpoint call. The better way is to make a single call to get db movies with collections and cross compare to that list.

Reply:
Yes, my bad, will take a look into it and do another commit, thanks.